docs/source/run.rst

```diff
@@ -57,7 +57,7 @@ Commonly Used Arguments
   Check the :ref:`faq` for ideas on tuning this parameter.
 * ``-k WORKERCLASS, --worker-class=WORKERCLASS`` - The type of worker process
   to run. You'll definitely want to read the production page for the
-  implications of this parameter. You can set this to ``$(NAME)``
+  implications of this parameter. You can set this to ``egg:gunicorn#$(NAME)``
```

Comment:
This change shouldn't be here :) See the original commit: https://github.com/benoitc/gunicorn/commit/8de5eb9b15bc429d6acece2935c60ec9f323d4f9

In general, the patch LGTM except this, but I can take care of it if you don't have time.

Thanks!